src/pages/index.js

```diff
@@ -105,13 +105,16 @@ export default function SiteIndex({ data }) {
           For Wardens
         </SecondaryNavItem>
       </SecondaryNav>
+
       {/* Hero */}
       <HomepageHero viewMode={viewMode} />
+
+      {/* Pizzazz */}
```

Comment:
I love that you felt it necessary to comment Pizzazz for the Pizzazz component, just in case we didn't know!